kubectl-openebs/cli/command/get/pool_list.go

```diff
@@ -87,8 +85,6 @@ func RunPoolsList(cmd *cobra.Command) error {
 		fmt.Println("No Pools are found")
```

Comment:
Need to decide to use one definitive way of the logging errors/warnings, klog or fmt.Println.

Please vote
❤️  for klog
🚀  for fmt.Println

Reply:
I looked into the [cobra library examples](https://github.com/spf13/cobra#using-the-cobra-library) and since we are following that, we can go with `fmt.PrintIn`.